.openshift-ci/build/push-images.sh

```diff
@@ -1,4 +1,51 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+registry_rw_login() {
+    if [[ "$#" -ne 1 ]]; then
+        die "missing arg. usage: registry_rw_login <registry>"
+    fi
+
+    local registry="$1"
+
+    case "$registry" in
+        docker.io/stackrox)
+            if [[ -z "$DOCKER_IO_PUSH_USERNAME" ]]; then
+                echo "DOCKER_IO_PUSH_USERNAME is not defined"
+                exit 0 # Change to exit 1 later
+            fi
+            docker login --username "$DOCKER_IO_PUSH_USERNAME" --password-stdin docker.io <<< "$DOCKER_IO_PUSH_PASSWORD"
+            ;;
+        quay.io/rhacs-eng)
+            if [[ -z "$QUAY_RHACS_ENG_RW_USERNAME" ]]; then
+                echo "QUAY_RHACS_ENG_RW_USERNAME is not defined"
+                exit 0 # Change to exit 1 later
+            fi
+            docker login --username "$QUAY_RHACS_ENG_RW_USERNAME" --password-stdin quay.io <<< "$QUAY_RHACS_ENG_RW_PASSWORD"
+            ;;
+        quay.io/stackrox-io)
+            if [[ -z "$QUAY_STACKROX_IO_RW_USERNAME" ]]; then
+                echo "QUAY_STACKROX_IO_RW_USERNAME is not defined"
+                exit 0 # Change to exit 1 later
+            fi
+            docker login --username "$QUAY_STACKROX_IO_RW_USERNAME" --password-stdin quay.io <<< "$QUAY_STACKROX_IO_RW_PASSWORD"
+            ;;
+        *)
+            echo "Unsupported registry login: $registry"
+            ;;
+    esac
+}
+
+shopt -s nullglob
+for cred in /tmp/secret/**/[A-Z]*; do
+    export "$(basename "$cred")"="$(cat "$cred")"
+done
+
 oc registry login
+
+registry_rw_login quay.io/rhacs-eng
+
+cd /go/src/github.com/stackrox/collector
+COLLECTOR_VERSION="$(make tag)"
+
+oc image mirror "$COLLECTOR_FULL" quay.io/rhacs-eng/collector:"$COLLECTOR_VERSION"
```

Comment:
Shouldn't we also push to `quay.io/stackrox-io`? This might be a good place to re-use `.circleci/images/100-push-images.sh` WDYT?

Reply:
It might not be possible to use push-as-manifest-list.sh, which is called by .circleci/images/100-push-images.sh. In the stackrox/stackrox version of push-as-manifest-list.sh there is the following line near the top of the script

```
[[ "${OPENSHIFT_CI:-false}" == "false" ]] || { die "Not supported in OpenShift CI"; }
```

The reason is that it might not be possible to create a manifest list with oc image. I asked Gavin about this and he does not seem to think that having manifest lists is important. I would like confirmation on this from other people.

There are other issues with using 100-push-images.sh. In the following
```
oc image mirror "$COLLECTOR_FULL" quay.io/rhacs-eng/collector:"$COLLECTOR_VERSION"
```
COLLECTOR_FULL is an environment variable created by Openshift CI, which is an image created by Openshift CI. If we have multiple collector images, we will have multiple environment variables for them. We need to be able to pair the Openshift CI locations for the images and the destinations we want for them. This would require a fairly major rewrite on 100-push-images.sh.